shared/actions/chat.js

```diff
@@ -298,8 +299,12 @@ function * _setupNewChatHandler (): SagaGenerator<any, any> {
 const followingSelector = (state: TypedState) => state.config.following
 
 function * _loadInbox (): SagaGenerator<any, any> {
-  // $ForceType
-  const inbox: GetInboxAndUnboxLocalRes = yield call(localGetInboxAndUnboxLocalRpcPromise, {params: {}})
+  const inbox: GetInboxAndUnboxLocalRes = ((yield call(localGetInboxAndUnboxLocalRpcPromise, {param: {
+    query: {
+      status: Object.keys(CommonConversationStatus).filter(k => !['ignored', 'blocked'].includes(k)).map(k => CommonConversationStatus[k]),
```

Comment:
Yeah... that's why I think we should add an "except status" to the query object? (So we don't have to get and pass a list of statuses that aren't ignored or blocked.) I guess that might be a thing we'd have to add to the protocol though.